config/config.exs

```diff
@@ -26,7 +26,12 @@ end
 
 
 config :smart_home_firmware,
-  hub: "192.168.0.14:4000"
+  hub: "192.168.0.14:4000",
```

Comment:
@th0mas assume this is a fixed/assigned address on your home network? Can we make it an environment variable so that its easier to run both on _my_ localhost and when it's deployed?

Reply:
Adding an environment variable could be a good option here 👍,

We could also use mDNS for deployment with Nerves later